server/sonar-server/src/main/java/org/sonar/server/webhook/WebHooksImpl.java

```diff
@@ -41,10 +42,12 @@
 
   private final WebhookCaller caller;
   private final WebhookDeliveryStorage deliveryStorage;
+  private final DbClient dbClient;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "dbClient" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)